lk2nd/device/dts/msm8974/samsung.dts

```diff
@@ -4,11 +4,45 @@
 #include <lk2nd.dtsi>
 
 / {
-	qcom,msm-id = <0xC208FF01 10 0x10000>,
+	qcom,msm-id = <0x7E01FF01 7 0>,
+		      <0xC208FF01 10 0x10000>,
 		      <0xC208FF01 14 0x10000>;
 };
 
 &lk2nd {
+	/* rev 07 */
+
+	hlte {
+		model = "Samsung Galaxy Note 3 (SM-N9005)";
+		compatible = "samsung,hlte";
+		qcom,msm-id = <0x7E01FF01 7 0>;
+		lk2nd,match-bootloader = "N9005*";
+
+		lk2nd,dtb-files = "msm8974-samsung-hlte";
+
+		gpio-keys {
+			compatible = "gpio-keys";
+			home {
+				lk2nd,code = <KEY_HOME>;
+				gpios = <&pmic 3 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
+			down {
+				lk2nd,code = <KEY_VOLUMEDOWN>;
+				gpios = <&pmic 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
+
+			up {
+				lk2nd,code = <KEY_VOLUMEUP>;
+				gpios = <&pmic 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
+		};
+
+		panel {
+			compatible = "samsung,s6e3fa0", "lk2nd,panel";
+			replace-compatible;
+		};
```

Comment:
Something got lost?

Reply:
Got rid off the downstream panels, leaving only the mainline one.

Reply:
If you don't need panel selection, drop the node please, the only use for panel {} is when you want to boot mainline and let lk2nd update the dt with the correct display compatible. (that is, when multiple display variants can exist in the same device model) If there is no subnodes, nothing will happen.

Reply:
Done.